src/Form/Type/AdminType.php

```diff
@@ -66,6 +66,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         $path = $this->getFieldDescription($options)->getFieldName().$options['property_path'];
                     }
 
+                    // Todo
+                    // Handle ParentFieldDescription, like contract.subscription.unitAccountSubscriptions
+                    // Handle PersistentCollection.
+
                     $subject = $p->getValue($parentSubject, $path);
```

Comment:
@rande Hi ! You're the one introducing the fix in order to handle case when builder has no data. I know it was in 2016, but maybe you can still help me.

In my case, I have a ContractAdmin, and a field
```
->add('subscription.unitAccountSubscriptions', CollectionType::class)
```

Here,
- `$options['property_path']` is `[0]`
-  `$this->getFieldDescription($options)->getFieldName()` is `unitAccountSubscription`

So it try to getValue `unitAccountSubscriptions[0]` and find nothing. It should use the `ParentFieldDescription` too, and look for something like `subscription.unitAccountSubscriptions[0]`.

I never used the PropertyAccessor...
If I write `$p->getValue($parentSubject, 'subscription.unitAccountSubscriptions[0]')`, it still doesn't work.
If I write `$p->getValue($parentSubject, 'subscription.unitAccountSubscriptions')`, it work but I get a collection.

Any suggestion ?